app/policies/ticket_policy.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class TicketPolicy
+  attr_reader :user, :ticket
+
+  def initialize(user, ticket)
+    @user = user
+    @ticket = ticket
+  end
+
+  def permitted_attributes_for_create
+    %i[sort price reservation_id seat]
+  end
+
+  def permitted_attributes_for_update
+    %i[id qr_code]
+  end
+
+  def permitted_attributes_for_validate
+    [:qr_code]
+  end
+
+  def index?
+    true
+  end
+
+  def create?
+    true unless user.role_ticket_checker?
```

Comment:
```suggestion
    !user.role_ticket_checker?
```

And similar  changes can be done in most of actions in this policy